jwst/lastframe/lastframe_step.py

```diff
@@ -7,28 +7,40 @@
 
 class LastFrameStep(Step):
     """
-    LastFrameStep: This is a MIRI specific task.  If the number of groups
-    is greater than 2, the GROUP data quality flags for the final group will
-    be set to DO_NOT_USE.
+    Set data quality in MIRI data of last group.
```

Comment:
Rephrase for clarity:
```suggestion
    Set data quality flags for the last group in MIRI ramps.
```